Validator/src/main/java/org/openzen/zenscript/validator/visitors/ExpressionValidator.java

```diff
@@ -96,7 +96,15 @@ else if (!expression.right.type.equals(expression.operator.getHeader().parameter
 	public Void visitCall(CallExpression expression) {
 		expression.target.accept(this);
 		checkMemberAccess(expression.position, expression.method);
-		checkCallArguments(expression.position, expression.method.method.getHeader(), expression.method.getHeader(), expression.arguments);
+
+		// ToDo: This probably needs to be done elsewhere?
+		//   Reason: INT_TO_STRING as method is (int): string
+		//   But the call is <int>.caster()
+		//   Hence the checkCallArguments method will say that () != (int) argument mismatch!
```

Comment:
<int>.caster() would be the right method header for this situation. To be checked why it isn't.